lib/cli.js

```diff
@@ -55,7 +56,11 @@ function run(argv) {
     });
   } else {
     files.forEach(function(file) {
-      formatToConsole(getSource(file), getConfig(file, argv));
+      if (argv.i) {
+        formatToSelf(file, getConfig(file, argv));
+      } else {
+        formatToConsole(getSource(file), getConfig(file, argv));
+      }
```

Comment:
It would be smarter for me to have two for each loops and only evaluate the conditional once, since it won't change.  A common compiler optimization; loop invariant code motion. ;)